test/unit/routes/claim/test-file-upload.test.js

```diff
@@ -95,7 +87,7 @@ describe('routes/claim/file-upload', () => {
 
   describe(`POST ${BASEROUTE}`, () => {
     it('should create a file upload object, insert it to DB and give 302', () => {
-      mockCallback.mockResolvedValue({})
+      // calls the 3rd arg of Upload, so the callback function
```

Comment:
Comment added so I don't spend hours trying to work out what it's doing to the stub 😆